test/rpc_test.py

```diff
@@ -291,11 +293,24 @@ def test_invalid_names(self):
         with self.assertRaisesRegex(RuntimeError, "shorter than 128"):
             rpc.init_model_parallel(
                 self_name="".join(["a" for _ in range(500)]),
-                backend=TEST_CONFIG.rpc_backend,
-                self_rank=self.rank,
-                init_method=self.init_method,
             )
+
+        from torch.distributed.rpc.api import _agent
+        self.assertEqual(_agent, None)
+        # join_rpc() should not do anything as _agent is None
         rpc.join_rpc()
```

Comment:
Do we need the join_rpc here if it doesn't do anything?

Reply:
no, it's not necessary, just to make sure it's fine to call join_rpc() even if init failed.